core/src/main/res/values/colors.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <!-- Colour Palette -->
```

Comment:
Are these still used?

Reply:
Yes, for the widget since it's not moved in compose with this PR. Furthermore, I'm afraid they can't be removed as the widget preview requires an xml layout even when moved to glance...